### ISSUE TITLE
Assign Quality based on Indexer group number

### ISSUE THREAD

**Issue by feerlessleadr** [lidarr/Lidarr#675]
**Is your feature request related to a problem? Please describe.**
Hopefully this is the right place to put this. I don't have a need for lossless music, so I generally search for albums in the 3010 group within newznab (for MP3). The problem I'm running into with Lidarr however, is that from what I can tell, most posts/releases in that specific indexer category don't specifically have an MP3 label on the release, so Lidarr labels the release as unknown, and doesn't pick it up.

Obviously this defeats the purpose for me, since I have to manually search and grab individual releases, rather than letting lidarr do all the work for me.

**Describe the solution you'd like**
Is it possible to have lidarr tag releases for quality based on both the filename, as well as the newznab group it is being pulled from?

**Describe alternatives you've considered**
Right now, I've removed all of the newznab categories from lidarr, with the exception of 3010, so that only mp3 results are returned, and have included the unknown quality in my lossy profile. 

**Additional context**

**Comment by Qstick** [comment]
The issue with this is that the category may tell us it’s MP3 but does not tell us the bitrate.

**Comment by feerlessleadr** [comment]
fair enough - is there anything I'm missing in terms of having lidarr automatically download mp3s?

I'm on a number of fairly popular indexing sites, and almost none of the mp3 releases are labeled as such by the uploader, so they all come up as unknown.

Thanks

**Comment by Qstick** [comment]
No, there are some things we could probably do get identification pre download a bit better. Just hasn't been a huge priority

**Comment by rouggy** [comment]
Hi all,
Having exact same issues, Lidarr cannot be used as such as all MP3 released are tagged as "unknown" and as such Lidarr does not download a single MP3 release. Indeed, the official MP3 releases do not include the MP3 tag in the title anymore...
This definitely defeats the purpose for me as well.
